### DISCUSSION TITLE
Usage documentation hard to find

### DISCUSSION THREAD

**Discussion by hughperkins** [steveicarus/iverilog#653] [Ideas]
the usage documentation can be found by doing the follwoing:
- going to icarus home page, http://iverilog.icarus.com/
- navigating to the iverilog github repo
- clicking on 'wiki', nestled amongst 'pull requests', 'actions', 'projects' etc
- clicking through to wikia
- clicking on 'user guide'
- cliking on 'user guide'
- Phew, arrived :) https://iverilog.fandom.com/wiki/User_Guide

Idea: put a link to https://iverilog.fandom.com/wiki/User_Guide directly in the iverilog home page, http://iverilog.icarus.com/
- also add a link to the README.md at the root of the iverilog github repo

**Comment by logicmonkey** [comment]
`man iverilog` works for me

**Reply by steveicarus** [reply]
I have recently put the documentation into the source code tree in github, and changed the github link to point to that code. It should be easier to find now.

**Comment by hughperkins** [comment]
Awesome :)
